app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -1487,4 +1494,25 @@ private void renderSubtitles(@Nullable final String text) {
             binding.subtitlesText.setVisibility(View.VISIBLE);
         });
     }
+
+    public boolean getServerVersionEqualOrGreater(@NonNull ServerVersion minServerVersion) {
+        Session session = sessionRepository.getValue().getCurrentSession().getValue();
+        if (session == null) {
+            Timber.d("couldn't compare server versions - session is null");
+            return false;
+        }
+
+        Server server = authenticationRepository.getValue().getServer(session.getServerId());
+        if (server == null) {
+            Timber.d("couldn't compare server versions - server is null");
+            return false;
+        }
+
+        boolean result = server.isVersionEqualOrGreater(minServerVersion);
+        Timber.d("current server version [%s] is %s [%s.%s.%s]",
+                server.getVersion(), result ? ">=" : "<",
+                minServerVersion.getMajor(), minServerVersion.getMinor(), minServerVersion.getPatch());
```

Comment:
You should be able to just do `minServerVersion.toString()` to get the version as "10.8.0"